apps/backend/src/modules/users/user.repository.ts

```diff
@@ -64,6 +64,7 @@ class UserRepository implements Repository {
 	}: PaginationQueryParameters): Promise<PaginationResponseDto<UserEntity>> {
 		const { results, total } = await this.userModel
 			.query()
+			.orderBy("createdAt", "desc")
```

Comment:
We have `SortType.DESCENDING`
```suggestion
			.orderBy("createdAt", SortType.DESCENDING)
```